release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -113,6 +113,13 @@ With this release, there are several updates to the *Administrator* perspective
 * The {product-title} web console displays a `ConsoleNotification` if the cluster is upgrading. Once the upgrade is done, the notification is removed.
 * A *_restart rollout_* option for the `Deployment` resource and a *_retry rollouts_* option for the `DeploymentConfig` resource are available on the *Action* and *Kebab* menus.
 
+[id="ocp-4-12-multi-arch-console"]
+===== Multi-architecture compute machines on the {product-title} web console
+
+The `console-operator` will now scan all nodes and build a set of all architecture types that cluster nodes run on and pass it to the `console-config.yaml`. The Operator can be installed on nodes with architectures of the values `amd64`, `arm64`, `ppc64le`, or `s390x`.
```

Comment:
s/will now scan all nodes and build/now scans all nodes and builds

You state _The Operator can be installed..._. Will the use know which Operator? It isn't clear to me (but I know I'm not the target audience.)

Reply:
Updated. Thanks for the suggestions!